<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestPassing</class>
 <widget class="QWidget" name="TestPassing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Прохождение теста</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Текст вопроса:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previousQuestion_pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Предыдущий вопрос</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>681</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="finishTest_pushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>430</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Закончить прохождение теста</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер вопроса:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextQuestion_pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>430</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Следующий вопрос</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Варианты ответов:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="questionSelect_comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>681</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
